### PR TITLE
Fix github workflow

### PR DESCRIPTION
**Pull request by wesleimp**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ jobs:
       - uses: actions/checkout@v1
 
       - name: Install OTP and Elixir
-        uses: ericmj/setup-elixir@emj/otp-branches
+        uses: actions/setup-elixir@v1
```
Comment (ericmj) [inline .github/workflows/main.yml:85]: The workflow is hosted here now https://github.com/erlef/setup-elixir/.

Comment (wesleimp) [reply .github/workflows/main.yml]: Nice

Comment (ericmj) [reply .github/workflows/main.yml]: Can you do the same for the format check? :)

Comment (wesleimp) [reply .github/workflows/main.yml]: Sorry I missed that. It's done

**Comment by ericmj** [comment]
Thank you 💜

**Pull request merged**